plugins/login-plugin/src/Components/UIComponents/Storefront/StoreFrontTv/EventDateTitle.js

```diff
@@ -0,0 +1,42 @@
+import React from "react";
+import { StyleSheet } from "react-native";
+import Label from "../../Label";
+import { mapKeyToStyle } from "../../../../Utils/Customization";
+import PropTypes from "prop-types";
+
+const EventDateTitle = (props) => {
+  const {
+    screenStyles,
+    payload: { extensions = {} },
```

Comment:
I'd recommend adding default props to avoid crash when the payload is missing:
https://reactjs.org/docs/typechecking-with-proptypes.html#default-prop-values